build.gradle

```diff
@@ -370,7 +370,7 @@ class TestsReportTask extends DefaultTask {
 }
 
 def graalLauncher = javaToolchains.launcherFor {
-    languageVersion.set(JavaLanguageVersion.of(17))
+    languageVersion.set(JavaLanguageVersion.of(rootProject.ext.buildJdkVersion))
```

Comment:
After checking further, I found that there were changes in GraalVM packaing. `lib/svm/bin/rebuild-images` and `lib/installer/bin/gu` do not exist in version 21.
Upgrading GraalVM needs to be handled as a separate issue.

Reply:
Interesting, I think in that sense graal builds should be optional based on JVM version. Otherwise it gave me an error regarding class version incompatibility.

Reply:
This might not be a required change, I think I got confused because of the warnings proguard threw. I hope it is gonna build after you revert the changes 👍